pets/meupet/models.py

```diff
@@ -92,7 +103,7 @@ class Pet(models.Model):
                                         help_text='Tamanho máximo da imagem é 8MB')
     published = models.BooleanField(default=False)
     created = models.DateTimeField(auto_now_add=True)
-    modified = models.DateTimeField(auto_now=True)
+    modified = models.DateTimeField(auto_now_add=True)
```

Comment:
I think `modified` will have the same value as the `created` attribute as both are using `auto_now_add`, take a look at the [docs](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateField.auto_now_add).